SSA/Experimental/Bits/Fast/Tactic.lean

```diff
@@ -0,0 +1,201 @@
+import Lean.Meta.Tactic.Simp.BuiltinSimprocs
+import SSA.Experimental.Bits.Fast.FiniteStateMachine
+import SSA.Experimental.Bits.Fast.BitStream
+import SSA.Experimental.Bits.Fast.Decide
+import SSA.Experimental.Bits.Lemmas
+import Qq.Macro
+
+open Lean Elab Tactic
+open Lean Meta
+open scoped Qq
+
+/-!
+# BitVec Automata Tactic
+There are two ways of expressing BitVec expressions. One is:
+
+
+```
+(x &&& y) || y + 0
+```
+
+the other way is:
+
+```
+Term.add (Term.or (Term.and (Term.var 0) (Term.var 1)) (Term.var 1)) Term.zero
+```
+
+The goal of this tactic is to convert expressions of the first kind into the second kind, because
+we have a decision procedure that decides equality on expressions of the second kind.
+-/
+
+
+
+section EvalLemmas
+variable {x y : _root_.Term} {vars : Nat → BitStream}
+def sub_eval :
```

Comment:
This should most likely be a `theorem`, not a `def`, since its type is a `Prop`.

Also, theorems are generally named according to the order *without* dot-notation.
This theorem statement is actually `eval (Term.sub _ _) vars`, so it should be names `eval_sub`.

Idem ditto for the next few theorems